Sources/RInAppMessaging/Router.swift

```diff
@@ -124,11 +142,90 @@ internal class Router: RouterType {
         return nil
     }
 
+    func displayTooltip(_ tooltipData: TooltipData,
+                        targetView: UIView,
+                        identifier: String,
+                        imageBlob: Data,
+                        becameVisibleHandler: @escaping (_ tooltipView: TooltipView) -> Void,
+                        completion: @escaping () -> Void) {
+
+        DispatchQueue.main.async {
+            self.displayedTooltips[identifier]?.removeFromSuperview()
+            self.displayedTooltips[identifier] = nil
+
+            let position = tooltipData.bodyData.position
+            let tooltipView = TooltipView(
+                model: TooltipViewModel(
+                position: position,
+                image: UIImage(data: imageBlob)!,
```

Comment:
What if the data passed in isn't valid image data?
Can we avoid this force unwrap?

Reply:
True, it shouldn't be like this. I missed that